controller/workitem.go

```diff
@@ -672,10 +713,14 @@ func (c *WorkitemController) ListChildren(ctx *app.ListChildrenWorkitemContext)
 		var response app.WorkItemList
 		application.Transactional(c.db, func(appl application.Application) error {
 			hasChildren := workItemIncludeHasChildren(ctx, appl)
+			converted, err := ConvertWorkItems(ctx.Request, wits, result, hasChildren)
```

Comment:
@kwk The `workItemIncludeHasChildren` function is hiding some potential errors. With earlier code that was difficult to capture as the `ConvertWorkItems` was _not_ returning error. But now with this refactoring, we can handle that. I think we can do that it in a separate PR. What do you think?

Reply:
We ARE handling errors now.

Reply:
(Comment pasted in chat) Please take a look at the `workItemIncludeHasChildren` function definition (line no. 636). The error from repo/model is not propagated.

Reply:
Okay now ;) 6055fb6